### PR TITLE
Antinoblium gas and supermatter reaction.

### PR DESCRIPTION
**Pull request by Pickle-Coding**
## About The Pull Request
Adds a gas reaction with antinoblium that allows nitrogen to combust to make nitrium. Exothermic reaction that releases an extreme amount of energy, radiation, zaps, and potentially emps. Antinoblium is used as a catalyst, and its potency increases with temperature, making it a hard to control reaction. If it releases too much energy, the zaps will become more dangerous and can release emp pulses. Cheaper compared to the traditional way of making nitrium, but more dangerous and requires a gas that is dangerous to make.

It reacts with the supermatter, having a heat penalty of 24, with a transmission multiplier of 90.

## Why It's Good For The Game
Antinoblium is probably the most inert gas in the game, despite being the antithesis of hypernoblium. It currently has absolutely no reaction with anything outside of the HFR. This will make it a more interesting gas to live up to its name, and adds more content for atmos to play with.

## Changelog

:cl:
add: Adds a new gas reaction. Requires nitrogen and oxygen, with antinoblium as a catalyst. Creates nitrium, but it also creates an extreme amount of energy.
add: The supermatter crystal will now react with antinoblium. Your engine will probably not be able to handle it.
/:cl:

### CODE REVIEW

**Comment by D4C-420** [comment]
going to put anti nob in the air and watch the station burn all for that sweet sweet nitruim.

**Comment by LemonInTheDark** [comment]
Drafted?

**Comment by Pickle-Coding** [comment]
I am waiting for the gas reaction cleanup PR to get merged or whatever, because then I would have to change most of my PR to comply with it.

**Comment by Pickle-Coding** [comment]
Outside of that, I'm going to do one small change (probably tomorrow) to reduce the hotspot flickering when there's a low amount of antinoblium.

**Comment by Pickle-Coding** [comment]
The gas reaction cleanup PR is getting stale so I'll open it anyways. Most of the changes I'd need to do would just be making defines.

File: code/modules/atmospherics/environmental/LINDA_fire.dm [hunk 1]
```diff
@@ -29,16 +29,20 @@
...
 	. = air_gases[/datum/gas/hydrogen]
 	var/h2 = . ? .[MOLES] : 0
+	. = air_gases[/datum/gas/antinoblium]
+	var/an = . ? .[MOLES] : 0
+	. = air_gases[/datum/gas/nitrogen]
+	var/n2 = . ? .[MOLES] : 0
 	if(active_hotspot)
 		if(soh)
-			if(plas > 0.5 || trit > 0.5 || h2 > 0.5)
+			if(plas > 0.5 || trit > 0.5 || h2 > 0.5 || (an > 0.5 && n2 > 0.5))
```
Comment (Ghilker) [inline code/modules/atmospherics/environmental/LINDA_fire.dm:38]: @LemonInTheDark this is why i wanted to generalize hotspots

Comment (Pickle-Coding) [reply code/modules/atmospherics/environmental/LINDA_fire.dm:38]: I was also thinking about that. We have cache_results for fire, so maybe we could base it on that instead? Wouldn't have to get a bunch of gases either.

**Comment by Pickle-Coding** [comment]
I will redo this after the gas reaction cleanup PR, and maybe change how hotspots works a little bit before doing this.

**Pull request closed without merging**